### PR TITLE
Update CUTENSORNET to use new slice group

### PR DESCRIPTION
**Pull request by kshyatt**
Stop using deprecated contraction function

### CODE REVIEW

File: lib/cutensornet/src/libcutensornet.jl [hunk 1]
```diff
@@ -484,8 +484,8 @@ end
     initialize_context()
     @ccall libcutensornet.cutensornetContractSlices(handle::cutensornetHandle_t,
                                                     plan::cutensornetContractionPlan_t,
-                                                    rawDataIn::Ptr{Ptr{Cvoid}},
-                                                    rawDataOut::Ptr{Cvoid},
+                                                    rawDataIn::Ptr{CuPtr{Cvoid}},
```
Comment (maleadt) [inline lib/cutensornet/src/libcutensornet.jl:487]: You also have to edit this in `res/wrap/cutensornet.toml`

**Review by maleadt — CHANGES_REQUESTED** [review]
LGTM, but the CuPtr change also needs to be added to the db or those changes will get lost.

File: lib/cutensornet/src/libcutensornet.jl (hunk 1 above)
Comment (kshyatt) [reply lib/cutensornet/src/libcutensornet.jl:487]: right sorry, will add

Comment (maleadt) [reply lib/cutensornet/src/libcutensornet.jl:487]: This is a recent change, you couldn't have known 🙂 Should probably add a "this is autogenerated" header, or something.

**Review by maleadt — APPROVED** [review]

**Pull request merged**